apps/webmin.sh

```diff
@@ -42,6 +42,16 @@ else
     print_text_in_color "$ICyan" "Installing Webmin..."
 fi
 
+# Check if local script is available
+if [ -f "$SCRIPTS"/apps/webmin.sh ]
+then
```

Comment:
Since this is happening frequently, maybe it would be better to make it a function instead?

Reply:
This happens (only) around 5 times, are you sure it makes sense?

Reply:
if you say yes, then I add it.